substrate/frame/support/src/traits/tokens/asset_ops.rs

```diff
@@ -0,0 +1,238 @@
+//! Abstract asset operations traits.
+//!
+//! The following operations are defined:
+//! * [`Inspect`]
+//! * [`Update`]
+//! * [`Create`]
+//! * [`Transfer`]
+//! * [`Destroy`]
+//! * [`Stash`]
```

Comment:
Shall the Lock/Unlock be added as well?

Reply:
On the other hand, it might be desirable to separate these operations (e.g., one pallet/contract may lock assets, and the other may only unlock).